<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>runner_view</class>
 <widget class="QWidget" name="runner_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>For all subjects of the current study :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="run_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stop_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="erase_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Erase results</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
